tensorflow/compiler/xla/service/hlo.proto

```diff
@@ -237,6 +243,9 @@ message HloInstructionProto {
   repeated xla.CustomCallOutputOperandAliasing
       custom_call_output_operand_aliasing = 74;
 
+  // Specifies the desired schedule for the custom-call.
```

Comment:
Specify that the field is only present for custom calls?

Reply:
Will add comments to make it clear.

Reply:
Added.